src/channels/channels.repository.ts

```diff
@@ -47,7 +47,8 @@ export class ChannelsRepository extends Repository<Channel> {
 			AND c."channelType" != 'PRIVATE' 
 			AND c."channelType" != 'DM'
```

Comment:
```suggestion
			AND (c."channelType" = 'PUBLIC' OR c."channelType" = 'PROTECTED')
```
저는 인덱스 생성을 고려해서 웬만하면 등호 연산자를 사용하는 편인데요 !
혹시 부정형을 사용하신 특별한 이유가 있는지 궁금합니다. 물론 channelType을 인덱스로 생성하진 않을 것 같습니다 . . .

Reply:
DM이랑 PRIVATE 채널 생성이 불가한걸 더 강조하고 싶었읍니당, 인덱스 생성고려는 안해봤는데 한수배워갑니다~